plasmapy/diagnostics/thomson.py

```diff
@@ -5,25 +5,38 @@
 
 __all__ = [
     "spectral_density",
+    "spectral_density_model",
 ]
 
 import astropy.constants as const
 import astropy.units as u
 import numpy as np
+import warnings
 
+from collections import namedtuple
+from lmfit import Model
 from typing import List, Tuple, Union
 
 from plasmapy.formulary.dielectric import permittivity_1D_Maxwellian
 from plasmapy.formulary.parameters import plasma_frequency, thermal_speed
 from plasmapy.particles import Particle
 from plasmapy.utils.decorators import validate_quantities
 
+# Define some constants
+C = const.c.si  # speed of light
```

Comment:
Speed of light seems to be the only constant used here, so instead of importing all constants and defining `C` just import `c`.

```python
from astropy.constants.si import c as C
```

or something similar